frontend/src/container/QueryBuilder/components/Query/Query.tsx

```diff
@@ -333,7 +333,7 @@ export const Query = memo(function Query({
 	const isVersionV4 = version && version === ENTITY_VERSION_V4;
 
 	return (
-		<Row gutter={[0, 12]}>
+		<Row gutter={[0, 12]} className={`query-builder-${version}`}>
```

Comment:
this is only for tests? we don't have a style for this.

Reply:
Yes, @SagarRajput-7, it's only for tests